lib/themes/dosomething/paraneue_dosomething/js/campaign/HotModule.js

```diff
@@ -1,362 +0,0 @@
-      if (this.chart.ctx.setLineDash) {
-        this.chart.ctx.setLineDash([5, 15]);
-      }
-      this.chart.ctx.stroke();
-
-      // clear dashes.
-      if (this.chart.ctx.setLineDash) {
-        this.chart.ctx.setLineDash([]);
-      }
-
-      // Draw a grid line at each of the highlighted data points.
-      Chart.helpers.each(this.scale.xLabels, function(label, index) {
-        if (data.highlighted.indexOf(label) > -1) {
-          // Check to see if line/bar here and decide where to place the line
-          const linePos = this.scale.calculateX(index - (this.scale.offsetGridLines ? 0.5 : 0)) + Chart.helpers.aliasPixel(this.scale.lineWidth);
-
-          ctx.beginPath();
-
-          ctx.lineWidth = this.scale.gridLineWidth;
-          ctx.strokeStyle = this.scale.gridLineColor;
-
-          ctx.moveTo(linePos, this.scale.endPoint);
-          ctx.lineTo(linePos, this.scale.startPoint - 3);
-          ctx.stroke();
-          ctx.closePath();
-
-          ctx.lineWidth = this.scale.lineWidth;
-          ctx.strokeStyle = this.scale.lineColor;
-        }
-      }, this);
-
-      // If we only have one point, then just draw that point on the graph.
-      if (onePoint) {
-        Chart.helpers.each(this.datasets[0].points, function(point) {
-          if (point.value !== null) {
-            ctx.beginPath();
-
-            ctx.arc(point.x, point.y, point.radius, 0, Math.PI * 2);
-            ctx.closePath();
-
-            ctx.strokeStyle = point.strokeColor;
-            ctx.lineWidth = point.strokeWidth;
-
-            ctx.fillStyle = point.fillColor;
-
-            ctx.fill();
-            ctx.stroke();
-          }
-        }, this);
-      }
-    },
-  });
-
-  // Actually instantiate the new graph and add it to the DOM.
-  const ctx = $el.get(0).getContext('2d');
-
-  // Calculate the point to scale the graph around.
-  const maxDataPoint = Math.max.apply(null, data.datasets[0].data);
-  const scaleMax = (goal > maxDataPoint) ? goal : maxDataPoint;
-
-  new Chart(ctx).lineWithGoal(data, {
-    showScale: true,
-    scaleOverride: true,
-    // Y-axis scale configuration.
-    scaleSteps: 10,
-    scaleStepWidth: scaleMax / 10,
-    scaleStartValue: 0,
-    scaleLineWidth: 1,
-    scaleShowLabels: false,
-    scaleFontFamily: '"Proxima Nova", "Helvetica Neue", "Helvetica", "Arial", sans-serif',
-    scaleFontStyle: 'Bold',
-    scaleFontColor: '#DDD',
-    scaleLineColor: 'rgba(255,255,255,.1)',
-    scaleFontSize: 14,
-    showTooltips: false,
-    scaleShowHorizontalLines: false,
-    scaleShowVerticalLines: false,
-    responsive: true,
-    bezierCurve: false,
-    pointDotRadius: 10,
-    pointDotStrokeWidth: 3,
-    datasetStrokeWidth: 5,
-    pointDot: false,
-  });
-};
-
-function init($chart = $('.js-progress-chart')) {
-  if (!$chart.length) return;
-
-  const progressData = setting('dosomethingCampaign.goals');
-
-  lineGraph($chart, progressData);
-}
-
-export default { lineGraph, init };
```

Comment:
totally. maybe we could have an archive repo for future reference if we want to do other charting stuff we have a ref to what was done before that isn't just this PR ;) food for thought :)